cmd/cmd.go

```diff
@@ -871,6 +871,17 @@ func NewCLI() *cobra.Command {
 	runCmd.Flags().Bool("insecure", false, "Use an insecure registry")
 	runCmd.Flags().Bool("nowordwrap", false, "Don't wrap words to the next line automatically")
 	runCmd.Flags().String("format", "", "Response format (e.g. json)")
+	runCmd.SetUsageTemplate(runCmd.UsageTemplate() + `
```

Comment:
It's not just run. `list`, `pull`, `push`, `create`, `cp`, any command that calls the API takes `OLLAMA_HOST`